recipes-core/util-linux/util-linux-nilrt.bb

```diff
@@ -1,4 +1,7 @@
-FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+LICENSE = "MIT"
```

Comment:
Why not just install the ptests in a `util-linux-ptest` subpackage?

Reply:
I considered it, but there's already one for util-linux.  Thought this would be equally good and not cause conflicts with the existing ptests.  If there's a better way I'm all ears.

Reply:
Well, we could try to make them play nicely together. But it's not a big deal, so dropped.